platform/src/main/java/org/stellar/anchor/platform/custody/fireblocks/FireblocksPaymentService.java

```diff
@@ -117,6 +140,16 @@ private CreateTransactionRequest getCreateTransactionRequest(JdbcCustodyTransact
         .build();
   }
 
+  private CreateTransactionRequest getCreateTransactionRefundRequest(JdbcCustodyTransaction txn)
+      throws InvalidConfigException {
+    return CreateTransactionRequest.builder()
+        .assetId(fireblocksConfig.getFireblocksAssetCode(txn.getAmountAsset()))
+        .amount(txn.getAmount())
+        // TODO: set source and destination
+        // TODO: what to do with amountFee/amountFeeAsset?
```

Comment:
`do_stellar_refund` doesn't have amountFeeAsset according to documentation:
![image](https://github.com/stellar/java-stellar-anchor-sdk/assets/131388170/5fef5ecb-8bc2-4aea-b4cd-221aabfdef8a)
But nevertheless, Custody Server needs to call `notify_refund_sent` when it receives payment notification. This action have `amount_fee` field. I think this is the place where we should use `amountFee` from `do_stellar_refund`